# global_frame: map
tf_prefix: dodo_detector_ros
detector_type: ssd
inference_graph: ~/.dodo_detector_ros/mscoco_frozen_inference_graph.pb
label_map: ~/.dodo_detector_ros/mscoco_label_map.pbtxt
ssd_confidence: 0.5
sift_min_pts: 10
sift_database_path: ~/.dodo_detector_ros/sift_database